eclipse-bluechi/workspace/workloads/agemo.kube

```diff
@@ -16,5 +16,8 @@
 Yaml=agemo.yml
 
 # Commented to disable the service to automatically start
-# [Install]
-# WantedBy=default.target
+[Install]
+WantedBy=default.target
+
+[Unit]
+Wants=ibeji.service
```

Comment:
Because Ibeji is configured to _use_ Agemo in the blueprint scenario, I would switch this dependency. 

Also, if we can have multiple in the list, this depends on chariott as well

For example, Agemo 
`Wants=mosquitto.service chariott.service`